### ISSUE TITLE
Behaviour always stays APPEND in the product import module.

### ISSUE THREAD

**Issue by Taral-Patoliya** [firegento/FireGento_FastSimpleImport2#14]
I need to import the products using add_update behavior and have set the settings in the system config the same, also i am adding the behavior in the command , but still the core model of import has behavior as APPEND, it does not change.

**Comment by EliasKotlyar** [comment]
Hello Taral-Patoliya,

Thanks for your Bug-Report. We havent figured out the difference between the two behaviors for now. Why do you insist on using the "add_update" behavior? Whats the difference? We have already one Issue for that question:
https://github.com/firegento/FireGento_FastSimpleImport2/issues/11


Greetings
Elias

**Comment by Taral-Patoliya** [comment]
@EliasKotlyar , Yes i have traced this thing and magento code has hard coded this condition in product import model. i thought that this may be affecting some of the productsthat i am importing Which i have already reported to them in [#6197](https://github.com/magento/magento2/issues/6197) and [#6193](https://github.com/magento/magento2/issues/6193), and they have acknowledged it and taged it as a bug.

**Comment by diglin** [comment]
Hi, I confirm the bug even. I work to find a solution even with the latest version of the firegento module at this time of writing.

**Comment by diglin** [comment]
Solution should come from Magento himself https://github.com/magento/magento2/issues/6193
